docs/server-release.md

```diff
@@ -9,7 +9,7 @@ To release the server:
 - [ ] Tag with `git tag VERSION`
 - [ ] Push version and tags, `git push && git push --tags`
 - [ ] Merge to `server-prod`:
-  - `git checkout server-prod && git merge master && git push`
+  - `git checkout server-prod && git merge -s ours master && git push`
```

Comment:
I think it's `-s recursive -X ours`.

Reply:
The comment https://github.com/mozilla/FirefoxColor/issues/368#issuecomment-401888654 links to the `ours` option for recursive, not the `ours` strategy.